### DISCUSSION TITLE
macOS port

### DISCUSSION THREAD

**Discussion by genericbar** [Tyrrrz/YoutubeDownloader#420] [General]
Hi, this is easily my favourite yt downloader app but unfortunately it's not available for macOS and I would love to equally be able to use it there. Is a macOS port up for consideration at all? 

Thanks for all the work!

**Comment by Tyrrrz** [comment]
As part of #295

**Reply by genericbar** [reply]
So it's in the works by another fellow member?

**Reply by Tyrrrz** [reply]
Yes

**Reply by genericbar** [reply]
Nice. Thanks for the great app!
